packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PeoplePickerItem.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+import { getId, classNamesFunction, styled, IStyleFunctionOrObject } from '../../../../Utilities';
+import { Persona, PersonaSize, IPersonaStyleProps, IPersonaStyles, IPersonaCoinStyleProps, IPersonaCoinStyles } from '../../../../Persona';
+import { IconButton } from '../../../../Button';
+import { ValidationState } from '../../BasePicker.types';
+import {
+  IPeoplePickerItemSelectedProps,
+  IPeoplePickerItemSelectedStyleProps,
+  IPeoplePickerItemSelectedStyles
+} from './PeoplePickerItem.types';
+import { getStyles } from './PeoplePickerItem.styles';
+
+const getClassNames = classNamesFunction<IPeoplePickerItemSelectedStyleProps, IPeoplePickerItemSelectedStyles>();
+
+export const PeoplePickerItemBase = (props: IPeoplePickerItemSelectedProps) => {
+  const { item, onRemoveItem, index, selected, removeButtonAriaLabel, styles, theme, className } = props;
+
+  const itemId = getId();
+
+  const classNames = getClassNames(styles, {
+    theme: theme!,
+    className,
+    selected,
+    invalid: item.ValidationState === ValidationState.warning
+  });
+
+  const personaStyles = classNames.subComponentStyles
+    ? (classNames.subComponentStyles.persona as IStyleFunctionOrObject<IPersonaStyleProps, IPersonaStyles>)
+    : undefined;
+
+  const personaCoinStyles = classNames.subComponentStyles
+    ? (classNames.subComponentStyles.personaCoin as IStyleFunctionOrObject<IPersonaCoinStyleProps, IPersonaCoinStyles>)
+    : undefined;
+
+  return (
+    <div
+      className={classNames.root}
+      data-is-focusable={true}
+      data-is-sub-focuszone={true}
+      data-selection-index={index}
+      role={'listitem'}
+      aria-labelledby={'selectedItemPersona-' + itemId}
+    >
+      <div className={classNames.itemContent} id={'selectedItemPersona-' + itemId}>
+        <Persona size={PersonaSize.size28} styles={personaStyles} coinProps={{ styles: personaCoinStyles }} {...item} />
```

Comment:
@joschect 
Having `{...item}` first means we are not allowing them to change the `size`, `styles` and `coinProps` in case they did. Do we wanna restrict custom changes of these props passed by the user and allow the rest of `IPersonaProps`?

Reply:
I think so? I'm not 100% on that, Either way is probably fine. I think overriding makes some amount of sense, but they can't override without also changing some picker styling. I'll leave this up to you.